### PR TITLE
Should be added to `dev` packages

### PR DESCRIPTION
**Pull request by eddiejaoude**
From the [docs](https://getcomposer.org/doc/03-cli.md#require)

> --dev: Add packages to require-dev

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ The Feature Browser tool generates .html files to represent your .feature files
 ## Installation
 Install via composer/packagist:
 ```
-composer require "feature-browser/feature-browser": "~1.0"
+composer require --dev "feature-browser/feature-browser": "1.0.0"
```
Comment (jenkoian) [inline README.md:9]: Personally I think leave the `--dev` bit out, leave it up to developer discretion (you could add a note about it I guess?). I also think `~1.0` is better than `1.0.0` or just omit the version number altogether and let composer work it out

File: README.md [hunk 2]
```diff
@@ -22,12 +22,10 @@ featurebrowser:
 ## Usage
 Run the featurebrowser generator from the command line:
 ```
-php bin/featurebrowser generate
+php bin\featurebrowser generate
```
Comment (jenkoian) [inline README.md:25]: Isn't this undoing your earlier PR?

Comment (eddiejaoude) [inline README.md:25]: This PR is out of date compared to master- let me update it.

File: README.md (hunk 1 above)
Comment (eddiejaoude) [inline README.md:9]: Yes, `~1.0` is better than `1.0.0`

File: README.md (hunk 2 above)
Comment (eddiejaoude) [inline README.md:25]: Actually no need if you do not want to use `--dev` (discussed above). I will close the PR.

**Pull request closed without merging**